runtime/jcl/common/dump.c

```diff
@@ -446,7 +451,7 @@ raiseExceptionFor(JNIEnv *env, omr_error_t result)
 
 	switch (result) {
 	case OMR_ERROR_INTERNAL:
-		exceptionClass = (*env)->FindClass(env, "com/ibm/jvm/InvalidDumpOptionException");
+		exceptionClass = (*env)->FindClass(env, "com/ibm/jvm/InvalidDumpOptionExceptionBase");
```

Comment:
Similar changes need to be made with respect to `DumpConfigurationUnavailableException`.

Reply:
Please correct me if I am mistaken, but `DumpConfigurationUnavailableException` is used only when we are changing the dump configuration, e.g. via `Java_com_ibm_jvm_Dump_setDumpOptionsImpl()`, so it isn't thrown by the trigger code.

Reply:
The native in `DiagnosticUtils` might pass `dumpOptions` that lead to `OMR_ERROR_NOT_AVAILABLE` if ever it is used.

Reply:
Fair enough.  I will fix it on Wednesday, along with any other changes you want.